practices/development.md

```diff
@@ -2,105 +2,152 @@
-It should not become so heavily-populated that it is hard to keep track of deliverables!
-:::
+  In this meeting we prioritize and assign the items that each team member will work on for this week, and review items that require discussion and planning.
 
-There are two Project Backlogs at 2i2c:
+Tues-Thurs
+: Team members work on the deliverables assigned to them at the sprint planning meeting.
 
-- The [2i2c Development Projects Backlog](https://github.com/orgs/2i2c-org/projects/7?fullscreen=true) contains deliverables for our development-focused projects.
-- The [Organizational Project Backlog](https://github.com/orgs/2i2c-org/projects/8?fullscreen=true) contains deliverables that are organization-wide and not related to development.
+Friday
+: By the end of the day, team members should have completed all of their deliverables for that week.
+  Each team member fills out a status update to share the major things they worked on, and note any unexpected challenges or blockers.
 
 (coordination:deliverables)=
 ## Deliverables
 
-Deliverables represent incremental amounts of value we can deliver to a particular stakeholder, and should be completable in a week or two.
+Deliverables represent incremental amounts of value we can deliver to a particular stakeholder.
+They are **encoded as GitHub Issues** and updated over time as we learn more about the particular deliverable.
+Most issues in our repositories are deliverables, in varying states of readiness.
 
-Most issues in our repositories are deliverables, in varying states of readiness. When a deliverable is first created, it may lack information, be improperly scoped, or have an unclear path to implementation. We improve this through _deliverable refinement_ (see below).
+### How are deliverables structured?
 
-A deliverable is ready to work (and can thus be added to the Project Backlog) when it has the following properties (adapted from the [INVEST methodology](https://agileforall.com/new-to-agile-invest-in-good-user-stories/)).
+There are a few special sections of a deliverable issue.
+Not all of them are strictly required, but are particularly useful for more complex or long-lasting deliverables.
 
-- **Have a short title and description:** Deliverables should be glanceable and have enough context that the reader can quickly understand the scope.
-- **Have one or more [user stories](https://www.atlassian.com/agile/project-management/user-stories):** User stories should define who benefits from the deliverable, and why they want it.
-- **Have completion criteria:** We have clear completion criteria for this deliverable to denote it as “done”.
-- **Have tasks to complete it**: Deliverables should have a set of tasks, which are actions needed to complete the deliverable.
+See [this Github Issue template](https://github.com/2i2c-org/team-compass/blob/main/.github/ISSUE_TEMPLATE/new-enhancement.md) for an example of a deliverable's structure.
+Below are some major sections that are common:
```

Comment:
I think that the structure should work for enhancements or admin stuff, though I suspect that bugs might benefit from its own dedicated issue structure (e.g. since it's often important to note environment details etc).

Maybe we should re-name this from `new-enhancement` to `new-deliverable`? We could use labels to track what kind of deliverable it was (e.g. admin, development, etc)